.travis.yml

```diff
@@ -61,27 +61,29 @@ matrix:
   - elixir: 1.9.1
     otp_release: 22.0.7
     env: WALLABY_DRIVER=selenium
-# Run in bigger container so we don't run out of memory generating the plt
-sudo: required
 cache:
   directories:
     - _build
     - deps
 
 dist: trusty
 
+addons:
+  apt:
+    sources:
+      - google-chrome
+    packages:
+      - dpkg
+      - google-chrome-stable
+  chrome: stable
+  firefox: latest
```

Comment:
Any reason why you chose chrome stable but firefox latest? I think using firefox stable may be less problematic

Reply:
OK I can see that we've been using latest before by downloading it manually. I think this may be a leftover from times when selenium webdriver wasn't well supported in Firefox but that's fine.

Reply:
Latest is the terminology it uses for the current version of firefox. 

https://docs.travis-ci.com/user/firefox/#version-aliases